cncnet-api/app/Http/Controllers/ApiLadderController.php

```diff
@@ -156,11 +153,8 @@ public function saveLadderResult($file, $ladderId, $gameId, $playerId, $pingSent
         if ($ladderId == 8 || $ladderId == 1) //toggle achievements on for Blitz and YR
             $this->updateAchievements($playerId, $history->ladder, $stats);
 
-        // Toggling off auto-wash, TODO only wash games where both players submitted a game report. 
-        // If only one player submitted a game report, don't wash
-
-        // if ($gameReport->best_report == true && $gameReport->duration == 3 && $gameReport->fps == 33)
-            // $this->adminService->doWashGame($gameReport->game_id, "ladder-auto-wash");
+        if ($gameReport->best_report == true && $gameReport->duration == 3 && $gameReport->fps == 33)
+            $this->adminService->doWashGame($gameReport->game_id, "ladder-auto-wash");
```

Comment:
this intended? @rohsyl 

Looks like a merge mistake, lines 156/157 should be commented out / removed

Reply:
remove or comment out

```
 if ($gameReport->best_report == true && $gameReport->duration == 3 && $gameReport->fps == 33)
            $this->adminService->doWashGame($gameReport->game_id, "ladder-auto-wash");
            ```

Reply:
fixed